Mugs/PortraitShiftTable.txt

```diff
@@ -0,0 +1,122 @@
+//// Portrait Upshift Edit(or?)
```

Comment:
I'm changing a lot of portrait IDs in another branch. Will that cause issues here?

Reply:
I was never able to get it to work. Will remove file.